packages/documentation-spa/blog/2021-03-10-state-of-one-platform-feb-2021.md

```diff
@@ -0,0 +1,189 @@
+---
+slug: state-of-one-platform-jan-2021
+title: State of One Platform - January 2021
+author: Ghanshyam Lohar
+author_title: Senior Software Engineer @ Red Hat
+author_url: https://github.com/ghanlohar
+author_image_url: https://avatars.githubusercontent.com/u/5575651?s=400&v=4
+tags: [redhat, blog, oneportal, oneplatform, react, angular, january, update]
+---
+### Meta
+**Completed Sprint:** One Platform – Feb '21
+
+**Ongoing/Upcoming Sprint:** One Platform – Mar'21
+
+**One Platform:** [one.redhat.com](https://one.redhat.com)
+
+**JIRA:** [https://projects.engineering.redhat.com/projects/ONEPLAT/issues](https://projects.engineering.redhat.com/projects/ONEPLAT/issues)
+
+**GitHub:** [https://github.com/1-Platform/one-platform/issues](https://github.com/1-Platform/one-platform/issues)
+
+### Introduction
+This is a summary of the development goals achieved by the One Platform team in the month of February 2021. This month the team had the following goals:
+
+- **Application Development:**
+  * Home SPA Documentation Hub to access all the document at one place
+  * Revamped pluggable Feedback SPA & Microservice development
+  * Automated end-to-end testing for One Platform initial set of modules
+  * DSAL stats dashboard & address user feedback
+  * Microservices’ enhancements(continued)
+  * Research Repository phase 2 features release
+
+- **Infrastructure update:**
+  * New strategy for using an external image repository to build and store One Platform Microservice docker images (either quay.io or GitHub container registry)
+
+- **Component Library:**
+  * Feedback component(opc-feedback) added
```

Comment:
```suggestion
  * Feedback component([opc-feedback](https://www.npmjs.com/package/@one-platform/opc-feedback)) added
```

Reply:
@riginoommen We have this link to the published component as part of the key highlights below(line#165). The thought behind this is We just want to put up a goal statement here.

Reply:
Okay